src/vargroups/VarGroupCollection.ts

```diff
@@ -32,9 +32,9 @@ export function displayValue(v: ValueType): string {
   if (v === undefined || isDeleted(v))
     return chalk.bgRed("     ");
   if (v === "")
-    return chalk.bgRedBright.black(` "" `);
+    return chalk.bgRedBright.black(" \"\" ");

```

Comment:
I don't like this change at all.  If the linter made you do this let's lose the lint rule.  Not even remotely worth it.

Reply:
I can make the rule allow backticks and double quotes, and then revert this back without warning. See here for the rule: https://eslint.org/docs/rules/quotes#allowtemplateliterals

I think linting of consistent quotes is very much worth. Your thoughts?

Reply:
Also ``` ` "" ` ``` looks a walrus 😁

Reply:
Yeah, I'm fine with that.